python/tests/stress/arcticdb/version_store/test_stress_append.py

```diff
@@ -87,17 +87,17 @@ def test_write_parallel_stress_schema_change(lmdb_version_store):
 
     random.shuffle(dataframes)
     for d in dataframes:
-        lmdb_version_store.write(symbol, d, parallel=True)
+        object_version_store.write(symbol, d, parallel=True)
 
-    lmdb_version_store.version_store.compact_incomplete(symbol, False, False)
-    vit = lmdb_version_store.read(symbol)
+    object_version_store.version_store.compact_incomplete(symbol, False, False)
+    vit = object_version_store.read(symbol)
     df.sort_index(axis=1, inplace=True)
     result = vit.data
     result.sort_index(axis=1, inplace=True)
     assert_frame_equal(vit.data, df)
 
 
-def test_write_parallel_stress_schema_change_strings(lmdb_version_store_big_map):
+def test_write_parallel_stress_schema_change_strings(basic_store_big_map):
```

Comment:
`big_map` is an LMDB specific concept. But think the idea is fine. Just calling the fixture `basic_store_large_data` or something would be fine I think.